samples/Azure.AI.DocumentTranslation/Generated/DocumentTranslationClient.cs

```diff
@@ -65,43 +65,33 @@ public DocumentTranslationClient(string endpoint, AzureKeyCredential credential,
         /// <param name="documentId"> Format - uuid.  The document id. </param>
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <remarks>
+        /// Below is the JSON schema for the request and response payloads.
```

Comment:
Let's only say request if we're including a sample for the request body, and only say response if we're including a sample for the response body.  Specifically:

- if there is only a request payload, please write 

    `Below is the JSON schema for the request payload.`

- if there is only a response payload, please write 

    `Below is the JSON schema for the request payload.`

- if we include schema for both request and response, please write 

    `Below are the JSON schema for the request and response payloads.`